src/topology/topological_fiber_bundle.lean

```diff
@@ -978,4 +979,49 @@ begin
   { exact A }
 end
 
+variable (i : ι)
+
+@[simp] lemma local_triv_coe : ⇑(Z.local_triv i) = Z.local_triv_ext i := rfl
+@[simp] lemma local_triv_source : (Z.local_triv i).source = (Z.local_triv_ext i).source := rfl
+@[simp] lemma local_triv_target : (Z.local_triv i).target = (Z.local_triv_ext i).target := rfl
+@[simp] lemma base_set_at : Z.base_set i = (Z.local_triv_ext i).base_set := rfl
+
+@[simp, mfld_simps] lemma local_triv_ext_apply (p : Z.total_space) :
+  (Z.local_triv_ext i) p = ⟨p.1, Z.coord_change (Z.index_at p.1) i p.1 p.2⟩ := rfl
+
+@[simp] lemma local_triv_at_apply (b : B) (a : F): ((Z.local_triv_at_ext b) ⟨b, a⟩) = ⟨b, a⟩ :=
+by { rw [local_triv_at_ext, local_triv_ext_apply, coord_change_self], exact Z.mem_base_set_at b }
+
+/-- The inclusion of a fiber into the total space is a continuous map.
+
+why does it find the sigma topology instead!? -/
+lemma continuous_sigma_mk (b : B) : @continuous _ _ _ (Z.to_topological_space ι)
+  (λ a, (⟨b, a⟩ : (bundle.total_space Z.fiber))) :=
```

Comment:
It the same statement is moved to `vector_bundle` it finds the correct topology on its own, but in this file it makes confusion.. why?

Reply:
@sgouezel ?

Reply:
Sorry, I had answered this one but clearly I have made a wrong manipulation. The reason it doesn't file the topology here is that it doesn't use your hint `⟨b, a⟩ : (bundle.total_space Z.fiber`. When Lean can find a unification solution on its own, it doesn't care about what you say, and it is pretty stubborn. A way out is to write `id ⟨b, a⟩ : (bundle.total_space Z.fiber`, but this is not really satisfactory. A better solution is to introduce a constructor for a point in the bundle, something like `Z.to_total_space b a`, with target type `Z.total_space`. I think it is a better solution, because otherwise this issue will pop up again and again.